sn_node_manager/src/bin/cli/main.rs

```diff
@@ -123,6 +124,22 @@ pub enum SubCmd {
         #[clap(long)]
         version: Option<String>,
     },
+    /// Start the safenode manager as a long running daemon
+    ///
+    /// This allows you to interact with the safenode manager through a RPC
+    Daemon {
```

Comment:
I feel like we should provide this as a subcommand. With respect to dealing with the faucet as a service, I've implemented that as a subcommand, so you use `faucet add`, `faucet start`, `faucet stop`, which keeps it consistent with node management. I think we should have the same thing here, because we should offer the user the ability to stop the daemon. Though actually stopping the daemon doesn't have to be implemented as part of this PR.

Reply:
Cool! I will refactor the code to accomdate that.